idaes/ui/flowsheet.py

```diff
@@ -259,6 +260,9 @@ def _map_edges(self):
         # Note we're only using the keys from self.ports, {port: parentcomponent}
         return set(self.ports) - used_ports
 
+    def _convert_for_ajax_compatibility(self, df):
+        return df.replace(np.nan, "NaN")
```

Comment:
Why not also to Inf and -Inf while we're at it. i.e. do the same with `np.inf` and `-np.inf`

Reply:
Done